contracts/clients/src/OperationResults.ts

```diff
@@ -28,23 +28,12 @@ export type OperationResult = {
   walletAddress: string;
   nonce: BigNumber;
   actions: ActionData[];
-  success: Boolean;
+  success: boolean;
   results: string[];
   error?: OperationResultError;
 };
 
-const getError = (
-  success: boolean,
-  results: string[],
-): OperationResultError | undefined => {
-  if (success) {
-    return undefined;
-  }
-
-  // Single event "WalletOperationProcessed(address indexed wallet, uint256 nonce, bool success, bytes[] results)"
-  // Get the first (only) result from "results" argument.
-  const [errorData] = results;
-
+export const decodeError = (errorData: string) => {
```

Comment:
I forgot to do this with `getOperationResults`, but would be nice to add a docstring describing this method.